osu.Game/Skinning/SkinManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Configuration;
+using osu.Framework.Platform;
+using osu.Game.Database;
+using osu.Game.IO.Archives;
+
+namespace osu.Game.Skinning
+{
+    public class SkinManager : ArchiveModelManager<SkinInfo, SkinFileInfo>
+    {
+        public readonly Bindable<SkinInfo> CurrentSkinInfo = new Bindable<SkinInfo>(SkinInfo.Default) { Default = SkinInfo.Default };

```

Comment:
We don't do this with any other component - `OsuGame.Beatmap` or `OsuGame.Ruleset`. Why is this in here?

Reply:
I'm thinking we want to go in this direction (and move `OsuGame`'s `Beatmap` into `BeatmapManager` too).